certified-connectors/tegolySIGN/readme.md

```diff
@@ -0,0 +1,12 @@
+## tegolySIGN Connector
```

Comment:
Please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for the readme file and for the raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). please include all required sections.